nixos/modules/programs/nist-feed.nix

```
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.nist-feed;
+in
+{
+  options = {
+    programs.nist-feed = {
+      enable = mkEnableOption (mdDoc "NIST Feed, which notifies you about the newest published CVEs");
+      package = mkPackageOption pkgs "nist-feed" { };
+      extraArgs = mkOption {
+        type = with types; listOf str;
+        default = [ "-l" "-s" "CRITICAL" ];
+        description = mdDoc ''
```

```suggestion
        description = ''
```
`lib.mdDoc` is no longer present in nixpkgs; please remove all usages.